CMakeLists.txt

```diff
@@ -101,9 +107,8 @@ else()
   set(LLVM_TARGETS_TO_BUILD ""
       CACHE STRING "Semicolon-separated list of targets to build, or \"all\".")
 
-
   # Then pull in LLVM for building.
-  add_subdirectory(${CLSPV_LLVM_SOURCE_DIR} EXCLUDE_FROM_ALL)
```

Comment:
Why is this necessary? It seems strange to add a directory in the build tree.

Reply:
In my case I checkout third_party/clspv, third_party/clspv-llvm and third_party/clspv-clang. You can't add a subdirectory which is outside your subtree without providing the build directory. So, to keep things the same I set the build directory to what it would have been if it was in-subtree.